__tests__/app.test.js

```diff
@@ -597,6 +598,51 @@ describe('PATCH /api/articles/:article_id', () => {
     });
 });
 
+describe('DELETE /api/comments/:comment_id', () => {
+    test('204: should have no content', () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(204)
+        .then(({ body }) => {
+            expect(body).toBeEmpty();
+        });
+    });
+
+    test('number of comments should decrease by one after successful deletion', () => {
+        return request(app)
+        .delete('/api/comments/5')  // comment 5 refers to article 1
```

Comment:
Please remove any comments before committing - your code should be readable enough that it doesn't need any comments :)